components/emailer/emailer.js

```diff
@@ -20,7 +20,12 @@ module.exports = class Emailer {
 
     const transport = transports[options.transport](options.transportOptions || {});
 
-    this.emailer = nodemailer.createTransport(transport);
+    if (options.transportOptions !== undefined &&
```

Comment:
The following would be much better here. Use lodash to get the attribute you need, if the attribute is undefined or null then it uses the transport variable instead. Much cleaner and refactors out all the unnecessary conditional code.
```
const emailerTransportOptions = _.get(options, 'transportOptions.host') || transport;
this.emailer = nodemailer.createTransport(emailerTransportOptions);
```